runtime/common/src/lib.rs

```diff
@@ -955,7 +966,7 @@ macro_rules! impl_config_traits {
             type ProposalBondMaximum = ProposalBondMaximum;
             type RejectOrigin = EnsureRootOrTwoThirdsCouncil;
             type SpendFunds = Bounties;
-            type SpendOrigin = NeverEnsureOrigin<Balance>;
+            type SpendOrigin = EnsureWithSuccess<EnsureRoot<AccountId>, AccountId, MaxTreasurySpend>;
```

Comment:
Can't find this one.

Reply:
- https://github.com/paritytech/substrate/blob/polkadot-v0.9.37/bin/node/runtime/src/lib.rs#L1111
- https://github.com/paritytech/substrate/blob/polkadot-v0.9.37/frame/system/src/lib.rs#L832-L854